### PR TITLE
chore(translations): update encryption text

### LINKED ISSUE #46: Chat - Add indication that messages are or are not encrypted.
They will be encrypted in 99% of cases but we should port over the message at the top of message history that says "Messages are secured by end-to-end Encryption and sent over a peer-to-peer network"

### PR DESCRIPTION
**Pull request by WanderingHogan**
### What this PR does 📖

Just updates the translations file with the text from the ticket, so it can be closed
English, Spanish (Luis), and Portuguese (Sara) added, will fall back on German
- 

### Which issue(s) this PR fixes 🔨

- Resolve #46

### CODE REVIEW

**Review by phillsatellite — APPROVED** [review]

**Comment by phillsatellite** [comment]
good when passes tests

**Review by dariusc93 — APPROVED** [review]

File: common/locales/es-MX/main.ftl [hunk 1]
```diff
@@ -54,6 +54,7 @@ messages = Messages
     .user-sent-message = te envió un mensaje.
     .unknown-sent-message = Alguien te envió un mensaje.
     .not-friends = Ya no son amigos
+    .msg-banner = Los mensajes están asegurados con encriptación de extremo a extremo y enviados a través de una red peer to peer
```
Comment (stavares843) [inline common/locales/es-MX/main.ftl]: this one misses a dot in the end compared with the others

Comment (stavares843) [inline common/locales/es-MX/main.ftl]:
```suggestion
    .msg-banner = Los mensajes están asegurados con encriptación de extremo a extremo y enviados a través de una red peer to peer.
```

File: common/locales/en-US/main.ftl [hunk 2]
```diff
@@ -54,7 +54,7 @@ messages = Messages
     .user-sent-message = sent you a message.
     .unknown-sent-message = someone sent you a message.
     .not-friends = You are not friends anymore
-    .msg-banner = Messages secured by local E2E encryption.
+    .msg-banner = Messages are secured by end-to-end Encryption and sent over a peer-to-peer network.
```
Comment (stavares843) [inline common/locales/en-US/main.ftl]: the other languages don't have Encryption in the first letter in caps lock

Comment (stavares843) [inline common/locales/en-US/main.ftl]:
```suggestion
    .msg-banner = Messages are secured by end-to-end encryption and sent over a peer-to-peer network.
```

**Review by stavares843 — APPROVED** [review]

**Pull request merged**